src/octoprint/plugins/softwareupdate/version_checks/github_release.py

```diff
@@ -140,6 +140,9 @@ def _get_base_from_version_tuple(version_tuple):
 	"""
 
 	base_version = []
+	# A leading v is common in github release tags. Remove it.
```

Comment:
This will only modify the version tuple of a base version ("1.2.0" instead of "1.2.0.dev23") was requested for version comparison - the `v` needs to be stripped from the tuple in any case however.

I think it would need to be moved into `_get_comparable_version_pkg_resources` for that.